test/build/10_sanity.bats

```diff
@@ -50,10 +50,20 @@ load ../common
             ch-checkns)
                 ;;
             *)
+                # CH_TEST_PHASE is an undocumented variable that allows us to
+                # execute ch-test with the automake without arguments.
+                local phase
+                if [[ -n $CH_TEST_PHASE ]]; then
+                    phase=$CH_TEST_PHASE
+                    unset CH_TEST_PHASE
+                fi
```

Comment:
It's unclear to me what adding this tests?

Reply:
This prevents the `executables sane` test from failing and bats from executing `ch-test all`. The documented and expected behavior for `./bin/ch-*` executables is print usage and exit. Without the above, `ch-test` will execute with the phase specified in `CH_TEST_PHASE` when executed with `make check`.

Reply:
Aha, OK. Would it work to simply `unset CH_TEST_PHASE`?